_data/simple-icons.json

```diff
@@ -9423,7 +9423,7 @@
         {
             "title": "Topcoder",
             "hex": "29A7DF",
-            "source": "http://topcoder.com/"
+            "source": "https://www.topcoder.com/thrive/articles/How%20to%20use%20the%20Topcoder%20GUI%20KIT"
```

Comment:
Since this page links to their brand guidelines, I think we use it for the `guidelines` entry too

Reply:
added.